src/components/VContentReport/VReportDescForm.vue

```diff
@@ -44,7 +44,7 @@ export default defineComponent({
      */
     reason: {
       type: String,
-      validator: (val) => Object.values(reasons).includes(val),
+      validator: (val) => reasons.includes(val),
```

Comment:
It would be nice to extract this kind of validation to a separate function to ensure it works well with TypeScript. The problem here is that `reasons` is a `const` array, and using `includes` with values that are not in that array throws an error: https://stackoverflow.com/questions/56565528/typescript-const-assertions-how-to-use-array-prototype-includes

Reply:
Good call, updated in 4b0b9ee.